python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -547,6 +547,31 @@ def test_will_item_be_pickled(lmdb_version_store, sym):
     assert_frame_equal(not_so_bad_df, lmdb_version_store.read(sym).data)
 
 
+def test_will_item_be_pickled_msgpack(lmdb_version_store_v1, sym):
+    lib = lmdb_version_store_v1
+
+    dict_data = {"a": [1, 2, 3], "b": {"c": 5}}
+    lib.write(sym, dict_data, recursive_normalizers=True)
+    assert lib.is_symbol_pickled(sym) == lib.will_item_be_pickled(dict_data, recursive_normalizers=True)
+    assert lib.will_item_be_pickled(dict_data) == True # msgpacked
+
+    list_data = [np.arange(3), np.arange(2 * 3), None]
+    lib.write(sym, list_data, recursive_normalizers=True)
+    assert lib.is_symbol_pickled(sym) == lib.will_item_be_pickled(list_data, recursive_normalizers=True)
+    assert lib.will_item_be_pickled(list_data) == True # msgpacked
+
+
+def test_will_item_be_pickled_recursive_normalizer(lmdb_version_store_v1, sym):
+    lib = lmdb_version_store_v1
+
+    list_data  = {"a": {"b": {"c": {"d": np.arange(24)}}}}
+    lib.write(sym, list_data, recursive_normalizers=True)
+    will_item_be_pickled = lib.will_item_be_pickled(list_data, recursive_normalizers=True)
+    assert lib.is_symbol_pickled(sym) == will_item_be_pickled
+    assert will_item_be_pickled == False
+    assert lib.will_item_be_pickled(list_data) == True
+
```

Comment:
You also need to handle the case where parts of the structure can be recursively normalized but parts would be pickled? Like,

```
dict_data = {"a": [1, 2, 3], "b": {"c": np.arange(24)}, "d": [CustomClass()]}
```

would we say that that would be pickled (correct) or wouldn't be (incorrect)